pkgs/development/compilers/odin/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, fetchFromGitHub
+, llvmPackages
+, makeWrapper
+}:
+
+let
+  inherit (llvmPackages) stdenv;
+
+  # Don't forget to change this when updating.
+  gitSha1 = "31989c9";
```

Comment:
Do we really need the short sha? We could also just inject the version to make updates easier.

Reply:
`odin version` outputs `<version>-<GIT_SHA>`. It would make no sense to set `GIT_SHA` to the version.
Do you a have a way to get the sha without `leaveDotGit`?

Reply:
No. We could just leave it empty to make it simpler.